admin_interface/models.py

```diff
@@ -17,19 +17,23 @@ class Theme(models.Model):
     @staticmethod
     def post_migrate_handler(**kwargs):
         del_cached_active_theme()
-        Theme.get_active_theme()
+        Theme.get_active_theme(**kwargs)
 
     @staticmethod
     def post_delete_handler(**kwargs):
         del_cached_active_theme()
-        Theme.get_active_theme()
+        Theme.get_active_theme(**kwargs)
 
     @staticmethod
     def post_save_handler(instance, **kwargs):
+
         del_cached_active_theme()
         if instance.active:
-            Theme.objects.exclude(pk=instance.pk).update(active=False)
-        Theme.get_active_theme()
+            objs_manager = Theme.objects
+            if "using" in kwargs:
+                objs_manager = objs_manager.using(kwargs["using"])
```

Comment:
Since I'll be replacing `*args,  **kwargs` could be replaced by `database="default", that can replace the `kwargs.get`. Will still be removing the if check.